src/cli/init/prompts.ts

```diff
@@ -66,6 +66,13 @@ const BASE_CHOICES = [
     validate: (value: unknown) =>
       isPlatform(value) || `must be ${PLATFORM_OPTIONS}`,
   },
+  {
+    name: 'defaultBranch',
+    message: 'Default Branch',
+    initial: 'master | main',
+    validate: (value: unknown) =>
+      typeof value === 'string' && value.length > 0 ? true : 'required',
```

Comment:
Do we still have annoying logic somewhere that stops the user from submitting `initial`? If not, I wonder if we should just default to `main` 🤔

Reply:
Yeah we do 😬 could work around that though

Reply:
...that looks like a rabbit hole, backing out!